### ISSUE TITLE
Management UI should indicate that a node is in maintenance mode

### ISSUE THREAD

**Issue by lukebakken** [rabbitmq/rabbitmq-server#8558]
### Is your feature request related to a problem? Please describe.

Currently, the only way to determine if a node is in maintenance mode is to use `rabbitmqctl status` and other CLI tools:

```
lbakken@shostakovich ~/development/rabbitmq/rabbitmq-server (main $%=)
$ ./sbin/rabbitmq-upgrade -n rabbit-1@shostakovich drain; echo $?
Will put node rabbit-1@shostakovich into maintenance mode. The node will no longer serve any client traffic!
0

lbakken@shostakovich ~/development/rabbitmq/rabbitmq-server (main $%=)
$ ./sbin/rabbitmqctl -n rabbit-1 status|grep -F maint
Is under maintenance?: true

lbakken@shostakovich ~/development/rabbitmq/rabbitmq-server (main $%=)
$ ./sbin/rabbitmqctl -n rabbit-1 --formatter=json status | jq -r .is_under_maintenance
true

lbakken@shostakovich ~/development/rabbitmq/rabbitmq-server (main $%=)
$ ./sbin/rabbitmqctl -n rabbit-1 --formatter=json cluster_status|jq .maintenance_status
{
  "rabbit-1@shostakovich": "넀[35mmarked for maintenance넀[0m",
  "rabbit-2@shostakovich": "not under maintenance",
  "rabbit-3@shostakovich": "not under maintenance"
}
```

It would be great to have an indication of this in the management UI.

### Describe the solution you'd like

The management UI "Overview" tab should show that a node is in maintenance mode. This could be indicated by a change in background color, similar to when the node is down.

### Describe alternatives you've considered

N/A

### Additional context

[VESC-1073](https://pivotal-esc.atlassian.net/browse/VESC-1073)

**Comment by SimonUnge** [comment]
Should it still show available stats in the overview, (file descriptors, socket desc etc), or just a banner with the status, i.e
<img width="1363" alt="image" src="https://github.com/rabbitmq/rabbitmq-server/assets/248800/cd349330-73c8-41d2-9d69-fed1367c70d6">

**Comment by michaelklishin** [comment]
I personally would prefer to see this either under the name (but not as a subscript, that would not be very visible) or in the Info column. A new column might work but we cannot guarantee that it will be displayed: the user controls what columns are shown.

So, a new "info item" in the Info column, in orange or another color different from light blue, to me sounds like the easiest option, and the optimal one at the same time.

**Comment by michaelklishin** [comment]
Closed by #8579 (contributed by @SimonUnge).
